__tests__/app.test.js

```diff
@@ -30,4 +30,32 @@ describe('/api/categories', () => {
             })
         });
     });
+});
+
+describe('/api/reviews', () => {
+    test('GET respond with an array of review objects.', () => {
+        return request(app)
+        .get('/api/reviews')
+        .expect(200)
+        .then((res) => {
+            expect(res.body).toEqual(expect.any(Array))
+            res.body.forEach((review) => {
+                expect(typeof review === 'object');
+            })
```

Comment:
I would say that this first assertion in a forEach is made redundant by the next